pkg/reconciler/identity/iam_policy.go

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package identity
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/iam"
+	"cloud.google.com/go/iam/admin/apiv1"
+	gclient "github.com/google/knative-gcp/pkg/gclient/iam/admin"
+	iampb "google.golang.org/genproto/googleapis/iam/v1"
+)
+
+type action = int
+
+const (
+	Add = iota
+	Remove
```

Comment:
```suggestion
	ActionAdd action = iota
	ActionRemove
```

In my experience these are normally prefixed with their 'type', e.g. `ActionAdd` and `ActionRemove`.

Reply:
Also, can they be private?